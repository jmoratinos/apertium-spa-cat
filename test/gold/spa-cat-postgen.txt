[DbnffY5_8krl]
per√≤ [/option]
[/DbnffY5_8krl]
