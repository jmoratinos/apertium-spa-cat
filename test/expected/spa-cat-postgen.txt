[DbnffY5_8krl#0] per√≤
[/DbnffY5_8krl]
